keyboards/ducky/one2mini/one2mini.c

```diff
@@ -0,0 +1,79 @@
+/* Copyright 2019 /u/KeepItUnder
+ * Copyright 2020 Reza Jelveh
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "one2mini.h"
+
+/**
+ * @brief   Board-specific bootmagic code.
+ * @details We want to avoid entering bootloader without holding two keys.
+ *          In this case the Ducky originals - "D" and "L".
+ *          These are D {2,3} and L {2,9}
+ *          We also need to avoid debounce by scanning twice with delay.
+ */
+void bootmagic_lite(void) {
```

Comment:
Does this add value over the default bootmagic_lite behaviour, it also doesnt implement an "and" per the functions comment.

Reply:
D+L are the default ducky bootloader enter keys. I think this might actually happen before the qmk boot in the original bootloader. But I didn't dare trying it out.

Reply:
Either way, my comment still stands. Having it be escape like most other boards has a lot of value when it comes to consistency, and it does not implement "hold both keys" like the comment suggests.

Reply:
removed